lib/tasks/spellcheck.rake

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'yaml'
+require 'rubygems' # import gem package manager
+require 'ffi/hunspell' # inject Hunspell class to Ruby namespace
+
+def extract_strings(data, extracted, prefix = '')
+  if data.is_a?(Hash)
+    data.each do |key, value|
+      extract_strings(value, extracted, prefix.empty? ? key : "#{prefix}.#{key}")
+    end
+  else
+    extracted << data
+  end
+end
+
+task spellcheck: :environment do
+  data = YAML.load_file('config/locales/en.yml')
```

Comment:
Can we load the `devise.en.yml` as well here? Is it possible to access the `config.i18n.load_path` and iterate over it so that we will always have all the I18n files loaded and spellchecked?